reference/manual-upgrade.md

```diff
@@ -0,0 +1,391 @@
+## Manual Upgrade
+
+Upgrading Mayastor from a version prior to 2.0.0 needs manual intervention. Follow the steps given below.
+To get started, take an etcd snapshot. The detailed steps for taking a snapshot can be found in the etcd [documentation](https://etcd.io/docs/v3.3/op-guide/recovery/).
```

Comment:
Mayastor uses etcd as a persistent datastore for its configuration. As a first step, we need to take a snapshot of the etcd.

Reply:
Updated the text.